### PR TITLE
adds additional paths to allow georouter

### PR DESCRIPTION
**Pull request by jsandland**
**Fixes following issues|Adds following features:**
- Allow georouter on additional pages beyond /express

**Resolves:** [MWPW-162338](https://jira.corp.adobe.com/browse/MWPW-162338)

**Steps to test the before vs. after and expectations:**
- Steps for feature 1

**Before**
**Georouter will not appear on these pages**
-  https://www.adobe.com/id_id/express/
-  https://www.adobe.com/de/express/
-  https://www.adobe.com/de/express/entitled
-  https://www.adobe.com/de/express/business
-  https://www.adobe.com/de/express/business/teams
-  https://www.adobe.com/de/express/nonprofits
-  https://www.adobe.com/de/express/learn/students
-  https://www.adobe.com/de/express//education/express
-  https://www.adobe.com/de/express/pricing

**Pages to check for regression and performance:**
**After**
**Georouter will now appear on these pages**
-  https://allow-georouter-on-additional-pages--express--adobecom.hlx.page/id_id/express/
-  https://allow-georouter-on-additional-pages--express--adobecom.hlx.page/de/express/
-  https://allow-georouter-on-additional-pages--express--adobecom.hlx.page/de/express/entitled
-  https://allow-georouter-on-additional-pages--express--adobecom.hlx.page/de/express/business
-  https://allow-georouter-on-additional-pages--express--adobecom.hlx.page/de/express/business/teams
-  https://allow-georouter-on-additional-pages--express--adobecom.hlx.page/in/express/nonprofits
-  https://allow-georouter-on-additional-pages--express--adobecom.hlx.page/in/express/learn/students
-  https://allow-georouter-on-additional-pages--express--adobecom.hlx.page/de/education/express/
-  https://allow-georouter-on-additional-pages--express--adobecom.hlx.page/de/express/pricing

### CODE REVIEW

File: express/scripts/utils.js [hunk 1]
```diff
@@ -2430,8 +2430,11 @@ async function loadPostLCP(config) {
     loadMartech();
   }
   const georouting = getMetadata('georouting') || config.geoRouting;
-  const isHomepage = window.location.pathname.endsWith('/express/');
-  if (georouting === 'on' && isHomepage) {
+  const permittedPaths = ['/express/', '/entitled',
```
Comment (JingleH) [inline express/scripts/utils.js:2433]: at this point, I would say maintaining this list in code would not be a good solution, so we should move to the metadata toggle approach like we do for jarvis, floating-cta, PEP, etc.

**Review by JingleH — CHANGES_REQUESTED** [review]

File: express/scripts/utils.js (hunk 1 above)
Comment (jsandland) [reply express/scripts/utils.js:2433]: Won't that mean that every document for a given path and the locale would need to have metadata added? I think that could get very  cumbersome. What about going by path but via spreadsheet like the existing locales? that way there is one place to make changes. What do you think?

Comment (JingleH) [reply express/scripts/utils.js:2433]: not really. I think I meant the same way as you said, using the spreadsheet bulk metadata approach. Our authors should already be very comfortable and familiar doing this. @vhargrave QQ: does Milo already have similar metadata authoring? If so then we can grab the key name directly

File: express/scripts/scripts.js [hunk 2]
```diff
@@ -14,7 +14,6 @@ import {
 const locales = {
   '': { ietf: 'en-US', tk: 'jdq5hay.css' },
   br: { ietf: 'pt-BR', tk: 'inq1xob.css' },
-  cn: { ietf: 'zh-Hans-CN', tk: 'puu3xkp' },
```
Comment (vhargrave) [inline express/scripts/scripts.js:17]: why are you removing this?

File: express/scripts/scripts.js [hunk 3]
```diff
@@ -29,7 +28,6 @@ const locales = {
   nl: { ietf: 'nl-NL', tk: 'cya6bri.css' },
   no: { ietf: 'no-NO', tk: 'aaz7dvd.css' },
   se: { ietf: 'sv-SE', tk: 'fpk1pcd.css' },
-  tr: { ietf: 'tr-TR', tk: 'ley8vds.css' },
```
Comment (vhargrave) [inline express/scripts/scripts.js:32]: and this?

File: express/scripts/utils.js (hunk 1 above)
Comment (vhargrave) [reply express/scripts/utils.js:2433]: yes, milo allows for that. We can turn off georouting over the config and just allow for `getMetadata('georouting')` to be set for the pages that want it. 

and yes, we probably would want this managed in the metadata spreadsheet, but @jsandland it looks like this should only be permitted for 8 pages or so? If it's only for 8 pages then authors should just do this in the word doc.

**Review by vhargrave — CHANGES_REQUESTED** [review]
These locales should not be removed, especially cn. 
I'd like to understand why you wanted to do it though?

File: express/scripts/scripts.js (hunk 2 above)
Comment (jsandland) [reply express/scripts/scripts.js:17]: Orla asked me to remove cn, tr and said that we do not have express sites in those locales.

File: express/scripts/scripts.js (hunk 3 above)
Comment (jsandland) [reply express/scripts/scripts.js:32]: Orla asked me to remove cn, tr and said that we do not have express sites in those locales.

Comment (JingleH) [reply express/scripts/scripts.js]: @jsandland I think we do though... https://www.adobe.com/cn/express/ are we looking to deprecate those locales? Then we might want a bigger discussion and lots of confirmation

**Review by vhargrave — CHANGES_REQUESTED** [review]
Hey Jackson! 
in general , I'd really suggest making as few changes to the scripts.js and especially the utils.js file. These files are already quite aligned with milo, so every change you do is breaking us away from our migration path.
Instead of a allowed-georouter-paths.json, authors should set the georouting metadata properly. If they need help with an easy way to do that, then we should provide support there, but not introduce new patterns which will not work with milo and make the page slower.

**Pull request closed without merging**